### PR TITLE
Move landing page to nextjs

### PR DESCRIPTION
**Pull request by igorosip0v**
[WID-227](https://linear.app/worldcoin/issue/WID-227/move-landing-page-to-nextjs)

Just a UI for now, the needed logic for these pages will be implemented later.

Vercel fails because we need to change vercel settings for new file structure

---
<details>
<summary>🎥 Result</summary>

https://user-images.githubusercontent.com/89008845/211896243-a0653a0d-efd5-4ec6-b3af-2094af855d04.mp4
</details>

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (maxpetretta) [inline public/manifest.json:2]:
```suggestion
  "short_name": "Discord Bouncer",
```

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "Create React App Sample",
```
Comment (maxpetretta) [inline public/manifest.json:3]:
```suggestion
  "name": "World ID Discord Bouncer bot",
```

File: public/site.webmanifest [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
```
Comment (maxpetretta) [inline public/site.webmanifest:3]:
```suggestion
    "short_name": "Discord Bouncer",
```

File: public/site.webmanifest [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
```
Comment (maxpetretta) [inline public/site.webmanifest:2]:
```suggestion
    "name": "World ID Discord Bouncer bot",
```

**Review by maxpetretta — APPROVED** [review]
Small naming changes can be made to manifests, but UI looks good to me

**Review by michaelfurutin — APPROVED** [review]

**Pull request merged**